scripts/convertBlacklistFormat.js

```diff
@@ -0,0 +1,20 @@
+const Account = require('../../models/account'); // temp
+const mongoose = require('mongoose');
+
+mongoose.Promise = global.Promise;
+mongoose.connect(`mongodb://localhost:27017/secret-hitler-app`);
+
+let count = 0;
+
+Account.find()
+	.lean()
+	.eachAsync(account => {
+		for (let i = 0; i < account.gameSettings.blacklist.length; i++) {
+			account.gameSettings.blacklist[i] = { userName: account.gameSettings.blacklist[i] };
```

Comment:
This should ensure the blacklist item is a string just to be safe

Reply:
yep, fixed!